application/client/src/app/service/session/dependencies/search/charts/request.ts

```diff
@@ -179,30 +184,66 @@ export class ChartRequest
                     event.on().state();
                 }
                 if (typeof desc.color === 'string' && this.set(true).color(desc.color)) {
-                    event.on().colors();
+                    event.on().color();
+                }
+                if (typeof desc.stepped === 'string' && this.set(true).stepped(desc.stepped)) {
+                    event.on().stepped();
                 }
                 if (
-                    typeof desc.background === 'string' &&
-                    this.set(true).background(desc.background)
+                    typeof desc.tension === 'string' &&
+                    this.set(true).tension(desc.tension)
                 ) {
-                    event.on().colors();
+                    event.on().tension();
+                }
+                if (
+                    typeof desc.borderWidth === 'string' &&
+                    this.set(true).borderWidth(desc.borderWidth)
+                ) {
+                    event.on().borderWidth();
+                }
+                if (
+                    typeof desc.pointRadius === 'string' &&
+                    this.set(true).pointRadius(desc.pointRadius)
+                ) {
+                    event.on().pointRadius();
```

Comment:
Why numeric props (`borderWidth`, `desc.pointRadius` etc.) are checked for `string` type? `typeof desc.borderWidth === 'string'`

Reply:
Can't find any specific reason for it, must've been a copy&paste mistake